browser/ui/webui/new_tab_page/branded_wallpaper.cc

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
nit: 2020?

Reply:
it was 2019 when I wrote that! 😆  I'll change, or remove in favor of `NTPSponsoredImagesData`

Reply:
💯2️⃣0️⃣2️⃣0️⃣💯